src/Illuminate/Log/LogServiceProvider.php

```diff
@@ -57,6 +57,10 @@ protected function channel()
      */
     protected function configureHandler(Writer $log)
     {
+        $logLineLevel = $this->logLineLevel();
```

Comment:
I suppose this can probably be inlined, `if ($logLineLevel = $this->logLineLevel()) {`